setup.py

```diff
@@ -53,25 +51,24 @@
     package_dir={"": "src"},
     include_package_data=True,
     zip_safe=False,
-    python_requires=">=2.7",
+    python_requires=">=3.8",
     install_requires=[
         "plone.app.registry",
+        "plone.app.upgrade",
```

Comment:
Please do not hard depend on `plone.app.upgrade` since this is something I and others usually do not want to have in my production instances (just in service instances).